packages/storybook-css/src/Alert.stories.tsx

```diff
@@ -72,12 +73,14 @@ export const OK: Story = {
 
 export const Warning: Story = {
   args: {
+    icon: <UtrechtIconCross />,
     type: 'warning',
   },
 };
 
 export const Error: Story = {
   args: {
+    icon: <UtrechtIconCross />,
     type: 'error',
```

Comment:
Om niet de indruk te wekken dat de icon hardcoded is, zou ik graag een `iconSet` dropdown willen hier. Als je op `iconSet` zoekt vindt je al snel andere stories die dit doen.

Reply:
Done voor de react en css stories